sessions_core/01_introduction.qmd

```diff
@@ -0,0 +1,751 @@
+
+::: {.look}
+Look at the environment panel on the top right, verifying that it is empty.
+:::
+
+::: {.write}
+Type the following command to save a variable called `cases` into your environment.
+
+```{r}
+#| eval: true
+cases <- 28
+```
+:::
+
+::: {.look}
+Look at the environment again. Is it still empty?
+:::
+
+If you’d like to access the value of your new object, `cases`, you simply need to **execute its name**.
+
+```{r}
+#| eval: true
+cases
+```
+
+::: {.callout-note}
+The reason we need to wrap strings in quotation marks is actually to allow R to differentiate between strings and object names.
+:::
+
+Once created, objects can be used in other commands:
+
+```{r}
+#| eval: true
+cases + 5
+```
+
+::: {.write}
+Create an object called `region` that contains the value `"Mandoul"`. Do you see it in your environment?
+:::
+
+```{r}
+#| include = solutions
+# SOLUTION
+region <- "Mandoul"
+```
+
+::: {.callout-tip}
+Don’t forget that we should always surround `<-` with spaces to improve readability and avoid errors.
+
+```{r}
+x<-3     # BAD
+x <- 3   # GOOD
+```
+:::
+
+### Updating an Object
+
+We often want to update the value stored in an object. To do this, we simply assign a new value with the same syntax we used to create it in the first place:
+
+```{r}
+cases <- 32
+```
+
+::: {.write}
+Update the value of `region` to `"Moyen Chari"`.
+:::
+
+```{r}
+#| include = solutions
+# SOLUTION
+region <- "Moyen Chari"
+```
+
+### Object Names
+When naming your objects, there are a few (relatively) hard rules:
+
+* Don’t start with a number
+* Don’t use spaces (use a `_` instead)
+* Don’t use protected values (like `TRUE` and `FALSE`) or function names (like `mean`)
+* Don’t use capital letters
+
+Beyond these hard rules, there are also more subjective best practices and personal styles. In general aim for names that are short and descriptive:
+
+```{r}
+a <- 19                             # BAD (not informative)
+age_du_patient_a_l_admission <- 19  # BAD (too long)
+age <- 19                           # GOOD
+```
+
+Giving your objects clear and informative names helps to make your code readable, making it easy for others to understand without the need for excessive documentation.
+
+
+## Data Structures
+
+Up until now we have looked only at simple objects that store single values, let’s now turn our focus to more complex structures that can store entire datasets.
+
+### Vectors
+We can collect multiple values (such as numerics or strings) into a single object, called a *vector*.
+
+Technically, there are several types of vectors, for example: 
+
+* *Simple vectors* (or *atomic vectors*) can only contain one type of values. For example, a numeric vector `2, 4, 6` or a string vector `"Mandoul", "Moyen Chari"`.
+* *Recursive vectors* (usually called *lists*) are far more complex and can contain multiple dimensions and types of data. We will not learn about them in this lesson.
```

Comment:
Do these get discussed at all in the course? If not, I'd suggest removing it, or at least to introducing it later. Again, if it's not needed now, then I think it's more manageable to take it out